docs/experimental-features.md

```diff
@@ -3,6 +3,9 @@
 > [!WARNING]
 > Experimental features should be enabled for testing purposes only, as there are no guarantees about the quality or stability of these features. Do not enable these settings for any production usage, or your production environment may be subject to breaking.
 
+> [!WARNING]
+> Experimental features in Bicep are not support by Microsoft customer support.
```

Comment:
This is related to the warning above it. Could you add this to the text, rather than adding a separate warning?